array.scad

```diff
@@ -91,43 +91,22 @@ module Cubic_and_Radial_Array_Test()
 
 
 // main lib modules
-module Cubic_Array(sx,sy,sz,nx,ny,nz,center)
-	{
+module Cubic_Array(sx,sy,sz,nx,ny,nz,center) {
 	if (center==true)
-		{
 		translate([-(((nx+1)*sx)/2),-(((ny+1)*sy)/2),-(((nz+1)*sz)/2)])
-			{
 			for(x=[1:nx])
-				{
 				for(y=[1:ny])
-					{
 					for(z=[1:nz])
-						{
 						translate([x*sx,y*sy,z*sz])
-						for (k = [0:$children-1]) child(k,center=true);;
-						}
-					}
-				}
-			}
-		}
+							children([0:$children-1],center=true);
```

Comment:
Hmm, I realize that `center=true` was there before, but I don't think that keyword argument does anything here.

Reply:
Also `children([0:$children-1])` can be shortened to just `children()`

Reply:
I disagree re children, the library are also example code and should be more standard usage than compressed efficiency, unless it is very important for performance.

Reply:
@Illusion65 I tried on the development snapshot with
```
module m() {
    children(0, center=true);
}

m() cube(10);
```
and that does not center as expected as `center` is not a valid argument for `children()`. Do you have a small example where it actually has an effect (and what OpenSCAD version)? I would consider that a bug in OpenSCAD. Maybe that worked in earlier versions by accident?

Reply:
Yes, you're right. I started playing around with the code a bit more. It's removed from the library.